services/121-service/src/events/utils/events.mapper.ts

```diff
@@ -41,10 +41,12 @@ export class EventsMapper {
 
   private static createAttributesObject(
     attributes: EventAttributeEntity[],
-  ): Record<string, any> {
+  ): Record<string, EventAttributeEntity['value']> {
     const attributesObject: Record<string, EventAttributeEntity['value']> = {};
     for (const attribute of attributes) {
-      attributesObject[attribute.key] = attribute.value;
+      if (attribute.value !== null) {
```

Comment:
This is a change in bevhaiour which probably makes sense but it should also be tested somehow (I'm not sure how)

Reply:
Agree, but when we start all different tests we can see. I think for now we can keep it like this.

Reply:
> when we start all different tests we can see

I'm not sure what you mean by this? Ideally this branch should be merged into main as is, so any relevant tests should already happen on this branch.